README.md

```diff
@@ -200,10 +200,8 @@ Note: The TLS settings here are for self-signed-certificates which are not recom
 
 ## Monitoring
```

Comment:
Documentation below should be updated, shouldn't it?

Reply:
Yep, missed it. Fixed in next commit.